queries.sql

```diff
@@ -9,3 +9,30 @@
  select * from animals where name !='Gabumon';
  select * from animals where weight_kg >=10.4 and weight_kg <=17.3;
 
+begin;
+update animals set species= 'unspecified';
+rollback;
```

Comment:
- [ ] According to the requirements, kindly ensure that after each transaction, you verify that the change was made, then you roll back the change and then verify that the change was undone. e.g:

``` UPDATE animals
   SET species = 'unspecified';  - make change

   SELECT species from animals; - verify that change was made
   ROLLBACK; - rollback change

   SELECT species from animals; - verify that change was undone
```